.github/workflows/pr-checks.yml

```diff
@@ -11,15 +11,15 @@ jobs:
 
     steps:
       - uses: actions/checkout@v1
-      - uses: Lundalogik/commitlint-github-action@master
+      - uses: wagoid/commitlint-github-action@v1
 
   autosquash:
     name: Block Autosquash Commits
     runs-on: ubuntu-latest
 
     steps:
       - name: Block Autosquash Commits
-        uses: Lundalogik/block-autosquash-commits-action@master
+        uses: xt0rted/block-autosquash-commits-action@v2.0.0
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@kickapoo As far as I understood, you said it should be safe to use third party actions, even though we give them the `secrets.GITHUB_TOKEN`, right?